tank/model_utils_tf.py

```diff
@@ -16,10 +16,48 @@
     # Invalid device or cannot modify virtual devices once initialized.
     pass
 
+################################## MHLO/TF models #########################################
+keras_models = [
+    "resnet50",
+]
+maskedlm_models = [
+    "albert-base-v2",
+    "bert-base-uncased",
+    "camembert-base",
+    "convbert-base-turkish-cased",
+    "deberta-base",
+    "distilbert-base-uncased",
+    "electra-small-discriminator",
+    "funnel-transformer",
+    "layoutlm-base-uncased",
+    "longformer-base-4096",
+    "mobilebert-uncased",
+    "mpnet-base",
+    "rembert",
+    "roberta-base",
+    "tapas-base",
+    "tiny-random-flaubert",
+    "xlm-roberta",
+]
+tfhf_models = [
+    "microsoft/MiniLM-L12-H384-uncased",
+]
```

Comment:
Maintaining this list seems irritating, is there a way it could be generated? Can be a TODO

Reply:
yes -- maybe we can pull it from tank/tf/tf_model_list.csv
I have added a TODO.